ibtk/src/lagrangian/LSiloDataWriter.cpp

```diff
@@ -149,7 +149,7 @@ build_local_marker_cloud(DBfile* dbfile,
 
     // Write out the variables.
     int cycle = time_step;
-    float time = float(simulation_time);
+    auto time = float(simulation_time);
```

Comment:
Can we use static_cast<float>() instead of float() here?

Reply:
Yes, although is it really better to write
```
auto time = static_cast<float>(simulation_time);
```
instead of
```
auto time = float(simulation_time);
```
?

The second one is certainly more compact.

Reply:
I guess the argument for the first one is to make it clear that type conversion is happening.

I guess I am pretty ambivalent in this case...